src/common/harvest/harvester.js

```diff
@@ -160,14 +172,12 @@ function send (agentRef, { endpoint, targetApp, payload, localOpts = {}, submitM
 
   function shouldRetry (status) {
     switch (status) {
-      case 429:
       case 408:
+      case 429:
       case 500:
-      case 503:
         return true
-      default:
-        return false
     }
+    return (status >= 502 && status <= 504) || (status >= 512 && status <= 530)
```

Comment:
This is a nit pick, feel free to ignore, but something feels weird to read for the mixing of the switch statement and the other check at the bottom.  It works so do what you want with that comment

Reply:
I know it sucks because listing out every code in this range would make the switch really large, but adding the single codes to an OR statement or if-else block would be very wordy too. Imma kick this can down